src/pks/flow/constitutive_relations/sources/distributed_tiles_evaluator.cc

```diff
@@ -102,8 +124,26 @@ DistributedTilesRateEvaluator::Update_(State& S)
 
     for (AmanziMesh::Entity_ID c = 0; c != ncells; ++c) {
       if (sub_marks[0][c] > 0) {
-        double val = std::min(p_enter_ - pres[0][c], 0.0) * dens[0][c] * k_;
+        if (th_ < 0){
+          //AmanziMesh::Entity_ID_List faces;
+          //std::vector<int> dirs;
+          const auto& [faces, dirs] = mesh->getCellFacesAndDirections(c);
+          double zmax = -1e+98;
```

Comment:
better practices would be to use:

`std::numeric_limits<double>::max` and
`std::numeric_limits<double>::lowest`. instead of +/-1e98.

Reply:
`#include <limits>`